src/users/04-moderation.md

```diff
@@ -31,3 +31,229 @@ All moderation actions are taken on the context menu of posts or comments. Click
 | Purge user         | Completely delete the user, including all posts and uploaded media. Use with caution.                                                       | Admin            |
 | Purge post/comment | Completely delete the post, including attached media.                                                                                       | Admin            |
 | Appoint as admin   | Gives the user administrator status                                                                                                         | Admin            |
+
+### Image Moderation
+
+> Attribution: This documentation was originally posted by Michael Altfield [here](https://tech.michaelaltfield.net/2024/03/04/lemmy-fediverse-gdpr/)
```

Comment:
Remove this as attribution is handled via git log. The docs are no place to advertise your website.

Reply:
Sorry, CC BY-SA requires attribution.

Reply:
This repo is under AGPL. We dont accept any other licenses for contributions.

Reply:
AGPL is compatible with CC BY-SA, so there is no issue re-licensing CC BY-SA content under the AGPL. And both licenses require attribution.